frontend/src/metabase/selectors/app.ts

```diff
@@ -202,9 +207,18 @@ export const getErrorMessage = (state: State) => {
 };
 
 export const getCollectionId = createSelector(
-  [getQuestion, getDashboard, getDashboardId],
-  (question, dashboard, dashboardId) =>
-    dashboardId ? dashboard?.collection_id : question?.collectionId(),
+  [
+    getQuestion,
+    getDashboard,
+    getDashboardId,
+    (state) => PLUGIN_DOCUMENTS.getCurrentDocument(state),
+  ],
+  (question, dashboard, dashboardId, document) =>
+    document
```

Comment:
I don't know if this is blocking, but the collection breadcrumbs logic is growing, and there may be a better way to organize it